apps/ml-service/venv/lib/python3.10/site-packages/dateutil/parser/__init__.py

```
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+from ._parser import parse, parser, parserinfo, ParserError
+from ._parser import DEFAULTPARSER, DEFAULTTZPARSER
```

Ruff (0.11.9)

`._parser.DEFAULTPARSER` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)